docs/_frontend/src/models/FilterForm/FilterForm.js

```diff
@@ -11,10 +11,36 @@ const FilterForm = ({ onSubmit, isLoading, meta, params }) => {
     ({ tasks, languages, editions } = meta.aggregations);
   }
   editions.sort((a, b) => {
-    if (a.indexOf('Healthcare') !== -1 && b.indexOf('Healthcare') === -1) {
-      return 1;
+    switch (a) {
```

Comment:
Looks complicated. We could make it much simpler.

Reply:
Great.. Thanks !